third_party/blink/renderer/modules/webaudio/audio_context.cc

```diff
@@ -799,6 +799,8 @@ void AudioContext::HandleAudibility(AudioBus* destination_bus) {
   }
 
   if (was_audible_ != is_audible) {
+    recordreplay::Assert("AudioContext::HandleAudibility %d %d", was_audible_, is_audible);
```

Comment:
I've updated with a comment, but I'm leaving it in as a tripwire in case there are future divergences in the audio system these changes don't address.

Reply:
Can you please always add issue numbers to asserts? Without it, we cannot auto triage it, and that will cause the person responsible for filtering (currently that's Kannan) unnecessary work.